tests/fixtures/angular-17/src/app/app.config.server.ts

```diff
@@ -0,0 +1,11 @@
+import { mergeApplicationConfig, ApplicationConfig } from '@angular/core';
+import { provideServerRendering } from '@angular/platform-server';
+import { appConfig } from './app.config';
+
+const serverConfig: ApplicationConfig = {
+  providers: [
+    provideServerRendering()
+  ]
+};
+
+export const config = mergeApplicationConfig(appConfig, serverConfig);
```

Comment:
I think we just want to extend the test matrix to include 17?
![image](https://github.com/netlify/angular-runtime/assets/3572990/0b6634fe-bc7e-42bc-b6cc-201d96eb69cb)

Reply:
What do you mean? Those are node versions, not Angular versions! 😄

Reply:
Evening brain, sorry